tests/io/duckdb/test_e2e.py

```diff
@@ -0,0 +1,74 @@
+from tempfile import TemporaryDirectory
+
+import duckdb
+import numpy as np
+import pandas as pd
+import pytest
+from minimalkv import get_store_from_url
+
+from plateau.io.duckdb.dataframe import (
+    read_table_as_ddb,
+    store_dataset_from_ddb,
+)
+from plateau.io.eager import store_dataframes_as_dataset
+
+
+@pytest.fixture
+def store_url():
+    dataset_dir = TemporaryDirectory()
```

Comment:
Don't use `TemporaryDirectory` but instead https://docs.pytest.org/en/stable/how-to/tmp_path.html#the-tmp-path-factory-fixture